examples/op-amp.jl

```diff
@@ -0,0 +1,174 @@
+# # International Space Station
+#md # [![](https://img.shields.io/badge/show-nbviewer-579ACA.svg)](@__NBVIEWER_ROOT_URL__/models/op-amp.ipynb)
+#
+#md # !!! note "Overview"
+#md #     System type: linear continuous system\
+#md #     State dimension: 2\
+#md #     Application domain: electrical engineering
+#
+# ## Model description
+#
+
+# Operational amplifiers ("op-amp") are electronic devices to amplify an input
+# voltage signal. They are widely used to construct filters that remove a
+# desired range of frequencies from the input signal [1].
+
+# The schematic diagram of an op-amp is shown below. It has two terminals:
+# - input side (left)
+# - output terminal (right)
+
+# ![](https://github.com/JuliaReach/ReachabilityAnalysis.jl/blob/master/examples/fig/opamp.png?raw=true)
+
+# The output voltage is ``e_o = K(e_B - e_A)``, where ``K`` is the voltage gain
+# of the op-amp. ``K`` is usually vey large (the order of ``10^5 V / V``).
+
+
+# In the *ideal op-amp* scenario, the following assumptions hold:
+# - The input terminals of the op-amp draw negligible current.
+# - The voltage difference at the input terminals ``e_B - e_A`` is zero.
+# - The gain ``K`` is infinite.
+
+# ## Example (inverting amplifier)
+
+# ![](https://github.com/JuliaReach/ReachabilityAnalysis.jl/blob/master/examples/fig/opamp_circuit.png?raw=true)
+
+# We can show that: ``e_o = -\dfrac{KR_2}{R_1 + R_2 + KR_1}e_{in}(t)``
+
+# In the ideal op-amp, $K\to \infty$ and ``e_O = -\dfrac{R_2}{R_1}e_{in}(t)``
+
+# ## Case of study (inverter with a capacitor)
+
+# ![](https://github.com/JuliaReach/ReachabilityAnalysis.jl/blob/master/examples/fig/opamp_c.png?raw=true)
+
+# We can show that the output satisfied the following ODE
+
+# ``\dfrac{de_o(t)}{dt} = -\dfrac{1}{R_2 C}e_o(t) - \dfrac{1}{R_1C}e_{in}(t)``
+
+# - Equation for the input voltage:
+
+# ``\dfrac{d \ ein}{dt} = \gamma \ ein + \delta``
```

Comment:
should we consider two more cases? 1) bounded input, constant, this is a special case of the one above with gamma and delta = 0 , and 2) bounded input but arbitrarily time-varying, u(t) in U

edit: just saw you also included 1) below

Reply:
Isn't case 2) the case with gamma and delta = 0 and ein(t=0)=U being U an interval?

Reply:
yes indeed. it's just that the solution method is sub-optimal because it could be using the `INT` solver since the system is effectively 1D. that said i agree that using the optimal settings is maybe not the main goal of these docs. how about adding this piece of information in a `note` block?